demisto_sdk/commands/content_graph/strict_objects/script.py

```diff
@@ -0,0 +1,87 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import (
+    SKIP_PREPARE_SCRIPT_NAME,
+    TYPE_PYTHON2,
+    TYPE_PYTHON3,
+)
+from demisto_sdk.commands.common.StrEnum import StrEnum
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    Argument,
+    BaseIntegrationScript,
+    BaseStrictModel,
+    CommonFields,
+    Important,
+    Output,
+    ScriptType,
+)
+
+
+class SkipPrepare(StrEnum):
+    SCRIPT_NAME = SKIP_PREPARE_SCRIPT_NAME
+
+
+class ScriptSubType(StrEnum):
+    PYTHON3 = TYPE_PYTHON3
+    PYTHON2 = TYPE_PYTHON2
+
+
+class ScriptArgument(Argument):
+    description_x2: Optional[bool] = None
```

Comment:
remove, just use `Argument` - and consider this comment for all models that only extend by adding a `_x2`